src/api/constants/field-ids/insurance/export-contract/index.ts

```diff
@@ -1,12 +1,15 @@
 import SHARED from '../../shared';
 
-const EXPORT_CONTRACT = {
+export const EXPORT_CONTRACT = {
   ...SHARED,
   ABOUT_GOODS_OR_SERVICES: {
     DESCRIPTION: 'goodsOrServicesDescription',
     FINAL_DESTINATION_KNOWN: 'finalDestinationKnown',
     FINAL_DESTINATION: 'finalDestinationCountryCode',
   },
+  HOW_WILL_YOU_GET_PAID: {
+    PAYMENT_TERMS_DESCRIPTION: 'paymentTermsDescription',
+  },
 };
 
 export default EXPORT_CONTRACT;
```

Comment:
Do we need second `default` export?

Reply:
Good question - at the moment yes.

In the main `field-ids` index file, we import via the default export.

But in E2E tests, we import via destructuring so without the default export.

Seems we should align this, but wouldn't be quick as should apply to all other field ID directories